app/src/main/java/com/example/concordia_campus_guide/ClassConstants.java

```diff
@@ -28,8 +29,11 @@
     public static final String DRIVING = "driving";
     public static final String SHUTTLE = "shuttle";
 
-    public static final String SGW_SHUTTLE_STOP = "1455 De Maisonneuve Blvd. W., SGW Campus";
-    public static final String LOYOLA_SHUTTLE_STOP = "7137 Sherbrooke St. W., Loyola Campus";
+    public static final Coordinates SGW_SHUTTLE_STOP = new Coordinates( 45.497025978950774, -73.57849985361099);
+    public static final Coordinates LOYOLA_SHUTTLE_STOP = new Coordinates(45.4584776424937, -73.638241700828);
+
+    public static final String ShuttlePolylineSGWLOY = "icutGlxa`MpCzBj@VNDv@T~BkHzC_JrAgEz@x@tJlJ`DvC~@|@`AtAf@_A\\u@l@_@ZIZAb@F\\N`@^tChEjHlL`A`BvDfHV`@FDTDdA~Bz@|An@jA|FnKfPfZ~ClFn@jA`@r@d@n@lBxBx@h@t@t@Zr@VXp@f@j@TbAXnAd@fCdAdAb@\\Xx@rAZb@Z\\v@r@x@t@x@nAn@zAZr@vBzCJLJBbD|ErApBJ\\@N?Jv@Zj@n@hDnFx@rAnApERZp@dBRb@vBjDfBrCVn@Rn@Nx@jBpJXpAZx@n@~AhAjCDh@Z|@Pt@Fb@?h@CtAEfA@h@Fr@f@jA`AvB~BfDHJPFr@zArAfDRf@Pr@^jBXrBnBfPj@rDb@fBNf@^bAPb@@Xb@hAQ\\sBjGuBhGeDnJI\\HPtDzInDhIzClHpBtEZj@lA`Dv@`BNXnAdBJTdDdE";
+    public static final String ShuttlePolylineLOYSGW = "armtGrmm`MtCjDbBpBhAnAp@l@`C`BnDjCvAqFj@gBvAeG|A}DLM{@kEy@uEoAuFa@iAsAaCq@kAY[a@iAIoAhBrA~@l@dCzAbBfAxDtBTDx@B~@Cf@Kb@Sx@k@R[HY?]EMi@_A_@u@yBgE[m@uAqBsAuBuEoHu@eA{AcCK[EKC]?OLOd@u@Va@DO@MAQEMgDsFsBkDQWkAaBkBeCqDmFWSOEeAuByA{Cc@}@cCyFuAyD_BwEiHsWu@eCs@cCSu@CCWo@_BmF{D_MiDeKwC}IeBoEiBmE}@cB{BkDy@aAs@y@_@c@c@[}BcB}@aAkCcBsCoAa@Q[_@m@q@_@]_@Sa@S_C}@_D}@{IeCiCw@eBq@oAs@qA{@{AoAwAyA[a@m@u@_@q@yAmCyAqCwCmEoAyBaLaTsHoN}BsDAa@_@y@wCqFo@mAeAsBW_@SWk@_@a@QaAc@]Uc@k@yDuFsBkCi@m@QO]Se@Is@@u@TSJc@Z[\\SVsBgBiF_FaIsHoCiCKXcAbCcCxGyBnFgB~EpAfA";
```

Comment:
what are those?

Reply:
if they are useful, can you leave a comment explaining them?

Reply:
Yeah I'll add one... These are the encoded polylines between both campuses since the route is always the same